docs/guide/overview-of-event-sources.md

```diff
@@ -131,6 +131,15 @@ functions:
     events:
       - sns: dispatch
 ```
+Or if you have a pre-existing topic ARN, you can just provide the topic ARN instead:
```

Comment:
Could you add the hint that the user should subscribe the function to the lambda here as well? Thanks!